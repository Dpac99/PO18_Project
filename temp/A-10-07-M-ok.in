100012
1
e1.obj
4
5
1
D1
P2
Sub 1
1
D1
P2
Sub 2
0
2
0
