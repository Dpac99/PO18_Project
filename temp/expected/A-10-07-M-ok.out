Identificador: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
0 - Sair
Escolha uma opção: Ficheiro a abrir: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
0 - Sair
Escolha uma opção: Opção inválida!
Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
0 - Sair
Escolha uma opção: Portal do Estudante
1 - Entregar projecto
2 - Preencher inquérito
3 - Ver resultados de inquérito
0 - Sair
Escolha uma opção: Nome da disciplina: Nome do projecto: Mensagem de entrega: Portal do Estudante
1 - Entregar projecto
2 - Preencher inquérito
3 - Ver resultados de inquérito
0 - Sair
Escolha uma opção: Nome da disciplina: Nome do projecto: Mensagem de entrega: Portal do Estudante
1 - Entregar projecto
2 - Preencher inquérito
3 - Ver resultados de inquérito
0 - Sair
Escolha uma opção: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
0 - Sair
Escolha uma opção: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
0 - Sair
Escolha uma opção: 